hail/src/main/scala/is/hail/linalg/BlockMatrix.scala

```diff
@@ -1660,18 +1657,35 @@ private class BlockMatrixMultiplyRDD(l: BlockMatrix, r: BlockMatrix)
         }
       })
 
+  def dgemm(c: BDM[Double], _a: BDM[Double], _b: BDM[Double]) {
```

Comment:
Nit, but I'd rather we called this something more like "fusedMultiplyAndAdd`. It really isn't `dgemm` unless it has a million more arguments, and we actually have `LAPACK.dgemm` in our codebase now for native arrays.